WikidataGame/WikidataGame.Backend/Migrations/DataContextModelSnapshot.cs

```diff
@@ -14,7 +14,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.2.6-servicing-10079");
+                .HasAnnotation("ProductVersion", "2.2.4-servicing-10062");
```

Comment:
Hm, ok I don't really know what to make of this (I know this is autogenerated but bear with me): The `ProductVersion` is lower and a lot of the migrations are deleted. Is that correct?

Reply:
I have honestly no clue regarding the product version 🤷‍♂️
During the last meeting I asked if we want to migrate old users and we agreed not to, therefore I decided to start off with a fresh migration. The changes we will apply because of the Firebase authentication would make a migration quite complex and would require additional UI within the unity game.

Reply:
I remember that discussion, you're right. But I was talking about something else:
![image](https://user-images.githubusercontent.com/525780/69491992-bace7100-0e9c-11ea-893d-1d73df427b16.png)
These migrations are not all user-related (I might have commented on the wrong file so that's not clear).

I just wasn't sure if all the data was still there when I saw the migrations were removed and the question about English kings was still in there. That was removed in #64 though which is not merged yet… could we get those two database states together somehow without losing either one?

Reply:
The migrations create the database cumulatively, in the order the migrations have been created. But the migrations are created from code. So when I delete all migrations and create a single new one I am not able to update an existing database anymore. But the newly created migration contains the whole structure and all seed data (categories and questions in our case) it is only missing the data which was inserted into the database through our API (I would consider that user data). So even if it looks strange with all the deletions, this does not delete changes to the database structure or seeds. 

Merging is therefore unproblematic as well, either we merge your PR first, then I will just have to merge the newest develop into my branch and recreate the migration. Or if we merge my PR first, you would have to merge develop into your branch and create an additional migration based on my initial one. So just a matter of a few seconds :)